packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -219,7 +219,7 @@ export function useInnerBlocksProps( props = {}, options = {} ) {
 					! isBlockSelected( clientId ) &&
 					! hasSelectedInnerBlock( clientId, true ) &&
 					enableClickThrough &&
-					! isDraggingBlocks(),
```

Comment:
Is it still necessary to have `isDraggingBlocks` given the addition of `isDragging`? When would one use one and not the other? Where do we still use `isDraggingBlocks`?

Reply:
It doesn't seem to be used much but it's a public API so we can't really get rid of it altogether 😅 
We could maybe refactor existing uses of it but that's better done as a separate PR I reckon. Not sure if there is a case for knowing that the thing being dragged is specifically a block 🤔